test/built-ins/Proxy/defineProperty/trap-is-undefined.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.6
+description: >
+    Return target.[[DefineOwnProperty]](P, Desc) if trap is undefined.
+info: >
+    [[DefineOwnProperty]] (P, Desc)
+
+    ...
+    8. If trap is undefined, then
+        a. Return target.[[DefineOwnProperty]](P, Desc).
+    ...
+---*/
+
+var target = {};
+var p = new Proxy(target, {});
+
+Object.defineProperty(p, 'foo', {
+    configurable: true,
+    enumerable: true,
+    writable: true,
+    value: 1
+});
+
+assert(target.hasOwnProperty('foo'), '`target` has own property `foo`');
+assert(target.propertyIsEnumerable('foo'), 'target.foo is enumerable');
+assert.sameValue(target.foo, 1);
+
+target.foo = 0;
+assert.sameValue(target.foo, 0, 'target.foo is writable');
+
+assert(delete target.foo, 'target.foo is configurable');
+
+Object.defineProperty(p, 'foo', {
+    configurable: false,
+    enumerable: false,
+    writable: false,
+    value: 2
+});
+
+assert.sameValue(target.propertyIsEnumerable('foo'), false,
+    'target.foo is not enumerable');
+assert.sameValue(target.foo, 2);
+
+target.foo = 0;
+assert.sameValue(target.foo, 2, 'target.foo is not writable');
+
+assert(delete target.foo, 'false', 'target.foo is not configurable');
```

Comment:
s/'false'/false/ and Reflect.deleteProperty otherwise a TypeError is thrown in strict mode.